### ISSUE TITLE
Service declaration is not compliant with symfony yml parsing

### ISSUE THREAD

**Issue by florentdestremau** [Hanfrey/sparkpost-bundle#5]
I get the following error:

```
[Symfony\Component\DependencyInjection\Exception\InvalidArgumentException]
The file "/var/www/preprod.windoo.fr/services/notification-service/shared/vendor/hanfrey/sparkpost-bundle/DependencyInjection/../Resources/config/services.yml" does not contain valid YAML.

[Symfony\Component\Yaml\Exception\ParseException]
The reserved indicator "@" cannot start a plain scalar; you need to quote the scalar at line 9 (near "arguments: [@hanfrey_sparkpost.client_adapter]")
```
So i manually add quotes but I'm surprised this could compile in the first place.

**Comment by florentdestremau** [comment]
See here:
- https://github.com/Hanfrey/sparkpost-bundle/blob/master/Resources/config/services.yml#L9
- https://github.com/Hanfrey/sparkpost-bundle/blob/master/Resources/config/services.yml#L17

**Comment by Hanfrey** [comment]
Fixed with #4
